src/pydocstyle/checker.py

```diff
@@ -673,6 +671,50 @@ def _check_numpy_section(cls, docstring, definition, context):
         if suffix:
             yield violations.D406(capitalized_section, context.line.strip())
 
+        if capitalized_section == "Parameters":
+            yield from cls._check_parameters_section(docstring, definition, context)
+
+    @staticmethod
+    def _check_parameters_section(docstring, definition, context):
+        """D417: `Parameters` section check for numpy style.
+
+        Check for a valid `Parameters` section. Checks that:
+            * The section documents all function arguments (D417)
+                except `self` or `cls` if it is a method.
+
+        """
+        docstring_args = set()
+        section_level_indent = leading_space(context.line)
+        content = context.following_lines
+        for current_line, next_line in zip(content, content[1:]):
+            # All parameter definitions in the Numpy parameters
+            # section must be at the same indent level as the section
+            # name.
+
```

Comment:
No need for so many blank lines between comments. But if you insist, put a comment on the empty lines too.